### DISCUSSION TITLE
예제실행방법 문의 todo-react-app node v17.2.0 에서

### DISCUSSION THREAD

**Discussion by dogfootman** [fsoftwareengineer/todo-application#18] [질문과 답변] [answered]
mac bicsur에서 

npm install 후
npm start 를 하면 실행이 되지않고 오류가 나고 있습니다.

버전을 낮춰서 실행해야 할까요??

> todo-react-app@0.1.0 start
> react-scripts start

node:internal/modules/cjs/loader:488
      throw e;
      ^

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './lib/tokenize' is not defined by "exports" in ....

**Comment by fsoftwareengineer** [comment]
node.js 12.16.1 LTS버전을 사용 해 주시기 바랍니다.

**Comment by fsoftwareengineer** [comment, marked as answer]
https://nodejs.org/dist/v12.16.1/ 에서 아래의 파일로 다운로드 하시면 될 것 같습니다. 
node-v12.16.1.pkg

**Comment by dogfootman** [comment]
오류나는 package가 postcss-safe-parser 였는데
npm install postcss-safe-parser  로 설치해서 해당오류 해결했습니다.
